### ISSUE TITLE
SDL2 example crashes

### ISSUE THREAD

**Issue by AileenLumina** [swistakm/pyimgui#71]
After fixing the typos described in #70, I ran into the following error when running [the SDL2 example](https://github.com/swistakm/pyimgui/blob/427ead566eea2ee20db685ab435b547a8417fa54/doc/examples/pysdl2.py) (Windows 10, latest SDL2.dll):

```python
Traceback (most recent call last):
  File "C:/Users/admin/PycharmProjects/BotW-World-Editor/main.py", line 103, in <module>
    main()
  File "C:/Users/admin/PycharmProjects/BotW-World-Editor/main.py", line 12, in main
    renderer = SDL2Renderer(window)
  File "C:\Users\admin\PycharmProjects\botwenv\lib\site-packages\imgui\integrations\sdl2.py", line 17, in __init__
    super(SDL2Renderer, self).__init__()
  File "C:\Users\admin\PycharmProjects\botwenv\lib\site-packages\imgui\integrations\opengl.py", line 90, in __init__
    super(ProgrammablePipelineRenderer, self).__init__()
  File "C:\Users\admin\PycharmProjects\botwenv\lib\site-packages\imgui\integrations\opengl.py", line 18, in __init__
    self._create_device_objects()
  File "C:\Users\admin\PycharmProjects\botwenv\lib\site-packages\imgui\integrations\opengl.py", line 115, in _create_device_objects
    last_array_buffer = gl.glGetIntegerv(gl.GL_ARRAY_BUFFER_BINDING)
  File "C:\Users\admin\PycharmProjects\botwenv\lib\site-packages\OpenGL\latebind.py", line 41, in __call__
    return self._finalCall( *args, **named )
  File "C:\Users\admin\PycharmProjects\botwenv\lib\site-packages\OpenGL\wrapper.py", line 664, in wrapperCall
    raise err
  File "C:\Users\admin\PycharmProjects\botwenv\lib\site-packages\OpenGL\wrapper.py", line 657, in wrapperCall
    result = wrappedOperation( *cArguments )
  File "C:\Users\admin\PycharmProjects\botwenv\lib\site-packages\OpenGL\error.py", line 232, in glCheckError
    baseOperation = baseOperation,
OpenGL.error.GLError: GLError(
	err = 1280,
	description = b'invalid enumerant',
	baseOperation = glGetIntegerv,
	pyArgs = (
		GL_ARRAY_BUFFER_BINDING,
		<object object at 0x000001E27D030110>,
	),
	cArgs = (
		GL_ARRAY_BUFFER_BINDING,
		<OpenGL.arrays.ctypesarrays.c_long_Array_1 object at 0x000001E27FA6C7C8>,
	),
	cArguments = (
		GL_ARRAY_BUFFER_BINDING,
		<OpenGL.arrays.ctypesarrays.c_long_Array_1 object at 0x000001E27FA6C7C8>,
	)
)

Process finished with exit code 1
```

**Comment by swistakm** [comment]
This is really interesting. Let's take care of #70 first and then I'll take a look at this one.

**Comment by swistakm** [comment]
Are you able to check your SDL version? I will dig into sources.

**Comment by AileenLumina** [comment]
I am using the 2.0.8 release build for Win32 x64 provided at [the official website](https://www.libsdl.org/download-2.0.php).

**Comment by AileenLumina** [comment]
Might be relevant to mention that, after removing

```python
if SDL_GL_SetSwapInterval(1) < 0:
        print("Warning: Unable to set VSync! SDL Error: ", SDL_GetError())
        exit(1)
```

from the example, the issue remains.

**Comment by WeavingBird1917** [comment]
Get Numpy and see if issue still remains, might be worth it just to check.

**Comment by swistakm** [comment]
I'm closing this because there was no follow-up. If anyone will have similar issue feel free to reopen.

**Comment by samkusin** [comment]
I experienced the same issue.  In my situation, I noticed my OpenGL version was being reported at 1.1 (which didn't make sense given I was on a GTX 1070 with up to date drivers.) 

Also I was using a RDP client for development purposes.  Apparently OpenGL and remote desktop via RDP do not work very well together and this is a known issue (DirectX on the other hand is fully supported in RDP.)

Switching to a native/host desktop setup resolved the issue for me, and it works beautifully.

I suppose someone could work on a DX9 port of the renderer for pyimgui.  I wish I had more time to do this though.
